# Generated by Django 2.2.5 on 2019-09-28 12:43

import django.contrib.gis.db.models.fields
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('garbage_map', '0003_auto_20190928_1116'),
    ]

    operations = [
        migrations.CreateModel(
            name='Event',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('lat_long', django.contrib.gis.db.models.fields.PointField(srid=4326)),
                ('start_time', models.DateTimeField()),
                ('city_name', models.TextField()),
                ('title', models.TextField()),
                ('url', models.URLField()),
                ('venue_name', models.TextField()),
                ('venue_address', models.TextField(null=True)),
                ('postal_code', models.TextField(null=True)),
                ('all_day', models.IntegerField(null=True)),
                ('calendar_count', models.TextField(null=True)),
                ('comment_count', models.TextField(null=True)),
            ],
        ),
        migrations.CreateModel(
            name='Weather',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('max_temp', models.FloatField()),
                ('mean_temp', models.FloatField()),
                ('mean_wind_speed', models.FloatField()),
                ('min_temp', models.FloatField()),
                ('total_prep', models.FloatField()),
                ('validdate', models.DateTimeField()),
            ],
        ),
    ]
